src/views/orb-view.ts

```diff
@@ -212,26 +249,20 @@ export class OrbView<N extends INodeBase, E extends IEdgeBase> implements IOrbVi
     }
   }
 
-  render(onRendered?: () => void) {
-    if (this._isSimulating) {
-      this._renderer.render(this._graph);
-      onRendered?.();
-      return;
-    }
-
+  private _assignPositions = (nodes: INode<N, E>[]) => {
     if (this._settings.getPosition) {
-      const nodes = this._graph.getNodes();
       for (let i = 0; i < nodes.length; i++) {
         const position = this._settings.getPosition(nodes[i]);
         if (position) {
           nodes[i].position = { id: nodes[i].id, ...position };
         }
       }
     }
+  };
 
-    this._isSimulating = true;
-    this._onSimulationEnd = onRendered;
-    this._startSimulation();
+  render(onRendered?: () => void) {
+    this._renderer.render(this._graph);
```

Comment:
Btw what happens in this situation:

```
const orb = new OrbView();
// This will call the simulator in web-worker thread
orb.setup({ very large graph })

// What happens now? When will recenter happen, after the simulation, during the simulation or?
orb.render(() => {
  orb.recenter();
});
```

Reply:
https://user-images.githubusercontent.com/3769376/212104176-a83af1de-c54c-4277-82af-3288e60128a3.mp4


https://user-images.githubusercontent.com/3769376/212104189-13967fe7-bcad-4741-b60c-7269b4ce4275.mp4